pkg/deviceshifu/deviceshifulwm2m/deviceshifulwm2mconfig_test.go

```diff
@@ -0,0 +1,54 @@
+package deviceshifulwm2m
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/edgenesis/shifu/pkg/deviceshifu/deviceshifubase"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateLwM2MInstructions(t *testing.T) {
+	tests := []struct {
+		name           string
+		input          *deviceshifubase.DeviceShifuInstructions
+		expected       *LwM2MInstruction
+		expectingPanic bool
+	}{
+		{
```

Comment:
add some test cases:
- instruction map is empty
- duplicate object id in different instruction
- objectId is empty or enableObserStr is empty(if we can test it)

Reply:
> add some test cases:
> 
> * instruction map is empty
> * duplicate object id in different instruction
> * objectId is empty or enableObserStr is empty(if we can test it)

The source code uses fatalf, which will terminate the program directly.